docs/src/main/asciidoc/_configprops.adoc

```diff
@@ -77,4 +78,4 @@
 |spring.cloud.consul.tls.key-store-password |  | Password to an external keystore.
 |spring.cloud.consul.tls.key-store-path |  | Path to an external keystore.
 
-|===
\ No newline at end of file
+|===
```

Comment:
Sorry for all the chatter and back and forth on this - having issue w/ formatting and not sure if my formatting is correct and it was previously checked in w/o trailing spaces removed.

Reply:
I have since learned that this is auto-generated in `spring-cloud-commons` - I am assuming the same here. I will remove my updates and move the docs to the javadoc of the properties